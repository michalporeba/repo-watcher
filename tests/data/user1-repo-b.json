{
  "account": "user1",
  "name": "repo-b"
}
