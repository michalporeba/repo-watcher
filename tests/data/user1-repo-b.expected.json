{
  "account": "user1",
  "owner": "user1",
  "name": "repo-b",
  "url": "https://github.com/user1/repo-b",
  "homepage": "(missing)",
  "default_branch": "main",
  "size": 222,
  "times": {
    "created": "2024-02-19T13:14:22Z",
    "updated": "2024-02-19T13:14:23Z",
    "pushed": "2024-02-28T18:44:08Z"
  },
  "blobs": {
    "description": "A test project by user1"
  },
  "counts": {
    "forks": 3,
    "openIssues": 5,
    "watchers": 8
  },
  "license": {
    "name": "(missing)",
    "spdxId": "(missing)"
  },
  "properties": {
    "isArchived": true,
    "isDisabled": true,
    "isFork": false,
    "isTemplate": true,
    "hasDiscussions": true,
    "hasDownloads": false,
    "hasIssues": true,
    "hasPages": true,
    "hasProjects": false,
    "hasWiki": true
  },
  "topLanguage": "Gleam",
  "languages": {},
  "workflows": {
    "total": 1,
    "active": 0
  }
}
